awd/wiley.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+headers = {
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+    "(KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"
+}
+
+
+def get_wiley_pdf(url, doi):
+    """Get PDF from Wiley server based on a DOI."""
```

Comment:
Adding a logging statement here will be helpful in debugging.

Reply:
I'm handling all logging in the integration function that you'll see in the next PR, but yes, this is one of the potential failure points that I'm tracking

Reply:
OK, I'll add that